pandas/core/frame.py

```diff
@@ -7057,7 +7065,7 @@ def value_counts(
             counts /= counts.sum()
 
         # Force MultiIndex for single column
-        if len(subset) == 1:
+        if is_list_like(subset) and len(subset) == 1:
```

Comment:
@mroeschke - this will make value_counts return a MultiIndex for any list-like (one element or not) but an Index (non-multi) for a single label. Wanted a 2nd opinion to make sure that is desirable behavior.

Reply:
I think that's reasonable behavior. Might be good to document that return difference